src/modules/context2d.js

```diff
@@ -2007,28 +2034,34 @@ import { console } from "../libs/console.js";
     sortPages(pages);
 
     var clipPath, oldSize, oldLineWidth;
-    if (this.autoPaging === true) {
+    if (this.autoPaging) {
       var min = pages[0];
       var max = pages[pages.length - 1];
       for (var i = min; i < max + 1; i++) {
         this.pdf.setPage(i);
 
+        var topMargin = (i === min ? this.posY + this.margin[0] : this.margin[0]);
```

Comment:
Should be i === 1 instead of i === min. options.text will contain one character instead of a block of text. This is what is causing the top margin bug mentioned.
```suggestion
        var topMargin = (i === 1 ? this.posY + this.margin[0] : this.margin[0]);
```